src/views/LoginView/LoginView.vue

```diff
@@ -39,9 +46,16 @@ export default {
         },
         changeModeLabel: function () {
```

Comment:
We generally use the syntax on the left:
`changeModeLabel () { ... } versus `changeModeLabel: function() {...}`